core/shared/src/main/scala/fs2/Pull.scala

```diff
@@ -464,16 +464,55 @@ object Pull extends PullLowPriority {
   }
 
   /* unrolled view of Pull `bind` structure * */
-
   private def viewL[F[_], O](stream: Pull[F, O, Unit]): ViewL[F, O] = {
 
+    def flatMapView[Q, P](fm: FlatMapOutput[F, P, Q]): ViewL[F, Q] = {
+      val st = Step[F, P](fm.stream, None)
+      new View[F, Q, Option[StepStop[F, P]]](st) {
```

Comment:
Is it possible/worth pulling out a named subtype of `View` that applies a continuation on success and fails on interrupted/fail?

Reply:
Done...